app/models/report_rendering_options.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class ReportRenderingOptions < ApplicationRecord
+  belongs_to :user, class_name: "Spree::User"
```

Comment:
I think we also should define this association on the User model. That way, we can set `dependent: :destroy`, ensuring that if a user is deleted, their settings will also be cleaned up. (eg https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/models/spree/user.rb#L32)